examples/NIMF-Metal-2-AWS-connection/terraform.tfvars.example

```diff
@@ -0,0 +1,22 @@
+equinix_client_id      = "<MyEquinixClientId>"
+equinix_client_secret  = "<MyEquinixSecret>"
+
+connection_name             = "Metal_2_AWS"
+connection_type             = "EVPL_VC"
+notifications_type          = "ALL"
+notifications_emails        = ["example@equinix.com"]
+bandwidth                   = 50
+purchase_order_number       = "1-323292"
+project_id                  = "<Project_ID>"
+aside_ap_authentication_key = "<Metal_Auth_Token>"
+aside_ap_type               = "METAL_NETWORK"
```

Comment:
If this is a metal network example, why is this an input that can be modified?

Reply:
We have added this example based on the Fabric Connection contract. We've included it as an input variable for testing purposes. Once the feature is ready for production(supported from Metal side as well), we will update it accordingly

Reply:
We should update this. The module has the variable as well but it will always be the same Access Point type for Metal Connection module, right?